test/test_dynamic_transform.cpp

```diff
@@ -1385,4 +1385,79 @@ TEST_F(NVFuserTest, ConcretizeConstantExtents) {
   testValidate(fusion, outputs, inputs, __LINE__, __FILE__);
 }
 
+// Test that dynamic reductions that should result in squeezes are handled
+// properly.
+// See https://github.com/NVIDIA/Fuser/issues/1667
+TEST_F(NVFuserTest, DynamicSqueezeTrivialReduction) {
```

Comment:
As a separate cleanup, consider s/NVFuserTest/DynamicTransformTest. This makes it easy to tell which file a test is in.